Content.Shared/Wires/WiresPanelSecurityLevelPrototype.cs

```diff
@@ -0,0 +1,17 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Wires
```

Comment:
file-scope namespace

Reply:
Not sure what the issue is here, using Robust.Shared.Prototypes is a requirement

Reply:
`namespace Content.Shared.Wires;` is preferable instead of


```
namespace Content.Shared.Wires 
{
}
```

Reply:
Oh OK. There are a lot of older files without file-scope namespacing though. E.g., AttemptElectrocute, AirlockSystem. Do I change those as well?

Reply:
Yeah everything should be file scoped, but many older files aren't yet. If you touch a file with the old scope there is no reason not to update it

Reply:
Thanks, will do then